installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -112,5 +112,13 @@
         "SLACK_TOKEN": "{slack_token}",
         "SLACK_WORKSPACE": "{slack_workspace}",
         "SLACK_SVC_ID": "{slack_svc_acc_id}"
+    },
+    "ACL": {
```

Comment:
Can we use the schema below? Basically, extending what you've got already to support other related configurations later. Set `ENABLED = true` by default.

```json
"ACL": {
  "ENABLED": true,
  "DATABASE": {
    "CLUSTER_READER_ENDPOINT": "",
    "CLUSTER_WRITER_ENDPOINT": "",
    "ENDPOINT": "",
    "PORT": "",
    "NAME": "",
    "CREDENTIAL_ID": ""
  }
}
```

Reply:
Done